spells.yml

```diff
@@ -5293,7 +5293,7 @@ infestation:
   components: V, S, M
   concentration: false
   description: |-
-    You cause a cloud of mites, fleas, and other parasites to appear momentarily on one creature you can see within range. The target must succeed on a Constitution saving throw, or it takes 1d6 poison damage and, if its speed is at least 5 feet, it moves 5 feet in a random direction. Roll a d4 for the direction: 1, north; 2, south; 3, east; or 4, west. This movement doesn't provoke opportunity attacks, and if the direction rolled is blocked, the target doesn't move.
+    You cause a cloud of mites, fleas, and other parasites to appear momentarily on one creature you can see within range. The target must make a Constitution saving throw. On a failed save, it takes 1d6 poison damage and, if its speed is at least 5 feet, it moves 5 feet in a random direction. Roll a d4 for the direction: 1, north; 2, south; 3, east; or 4, west. This movement doesn't provoke opportunity attacks, and if the direction rolled is blocked, the target doesn't move. On a successful save, it takes half as much damage and does not move.
```

Comment:
Contractions are typically used:
`On a successful save, the target takes half as much damage and doesn't move.`